src/js/view/controls/controls.js

```diff
@@ -408,7 +408,10 @@ export default class Controls {
         api.setMute(mute);
         // the model's mute value may not have changed. ensure the controlbar's mute button is in the right state
         this.controlbar.renderVolume(mute, model.get('volume'));
-        this.mute.hide();
+        if (OS.mobile) {
+            this.mute.hide();
+        }
```

Comment:
Instead we should guard with `if (this.mute) {`. That was it doesn't matter if we change which env we add or don't add the mute button.